CODE_OF_CONDUCT.md

```diff
@@ -65,7 +65,7 @@ Representation of a project may be further defined and clarified by project
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-  reported by contacting [cpp-conduct@nvidia.com](mailto:cpp-conduct@nvidia.com).
+  reported by contacting [cpp-conduct@nvidia.com].
```

Comment:
The link for [Contributor Covenant version 1.4] is missing at the bottom of this page.

Reply:
Done.